index.js

```diff
@@ -57,34 +53,50 @@ app.post('/webhook', function(req, res) {
 });
 
 function receivedMessage(event) {
-  var senderID = event.sender.id;
-  var recipientID = event.recipient.id;
-  var timeOfMessage = event.timestamp;
-  var message = event.message;
+  const senderID = event.sender.id;
+  const recipientID = event.recipient.id;
+  const timeOfMessage = event.timestamp;
+  const message = event.message;
 
   console.log("Received message for user %d and page %d at %d with message:",
     senderID, recipientID, timeOfMessage);
   console.log(JSON.stringify(message));
 
-  var messageId = message.mid;
+  const messageId = message.mid;
 
-  var messageText = message.text;
-  var messageAttachments = message.attachments;
+  const messageText = message.text;
+  const messageAttachments = message.attachments;
 
   if (messageText) {
 
     // If we receive a text message, check to see if it matches a keyword
     // and send back the example. Otherwise, just echo the text we received.
     switch (messageText) {
-    case 'generic':
-      sendGenericMessage(senderID);
-      break;
-    case 'hello world':
-      sendTextMessage(senderID, "hello to you too");
-      break;
-
-    default:
-      sendTextMessage(senderID, messageText);
+      case 'laundry':
+        $.getJSON('https://api.pennlabs.org/laundry/halls', function (response) {
+          let hallsArray = response['halls'];
+          for (let i = 0; i < hallsArray.length; i++) {
+            let dryers_available = hallsArray[i]['dryers_available'];
+            let name = hallsArray[i]['name'];
+            let washers_available = hallsArray['washers_available'];
+
+            let ret = name.concat(": There are ").concat(dryers_available).concat(" available dryers and ")
```

Comment:
const ret = `${name}: There are ${dryers_available} available dryers and ${washers_available} washers available!`;